conda-forge.yml

```diff
@@ -1,3 +1,6 @@
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```

Comment:
That said, it is ok to include both here as we did in the Arch Migrator PR ( https://github.com/conda-forge/cuda-nsight-feedstock/pull/1 )

If this feedstock were to add `linux_aarch64`, we would be able to use cross-compilation to start